src/routes/cube/index.js

```diff
@@ -1027,7 +1027,7 @@ router.get('/samplepackimage/:id/:seed', async (req, res) => {
     });
 
     res.writeHead(200, {
-      'Content-Type': 'image/webp',
+      'Content-Type': 'image/jpeg',
```

Comment:
This is only functional change in the file. The rest were update by prettier/eslint